src/v/storage/backlog_controller.cc

```diff
@@ -10,8 +10,8 @@
 
 #include "config/configuration.h"
 #include "prometheus/prometheus_sanitize.h"
-#include "ssx/sformat.h"
 #include "ssx/future-util.h"
+#include "ssx/sformat.h"
```

Comment:
nit: You could probably have been tidier in the previous commit for some of these.